.gitignore

```diff
@@ -301,3 +301,4 @@ Resources/MapImages
 /Content.Docfx/api/
 /Content.Docfx/*site
 
+/Content.Server/Properties/launchSettings.json
```

Comment:
what is this for? vscode stuff?

Reply:
Yeah that snuck in there. Thought I removed it, just local settings.